docs/datafeed.md

```diff
@@ -0,0 +1,127 @@
+# Datafeed
+
+The Datafeed Service is a service used for handling the [_Real Time Events_](https://developers.symphony.com/restapi/docs/real-time-events). 
+For each interaction of a user within the IM, MIM or Room chat, an event will be sent to the datafeed.
+The bot can create a datafeed, list all created datafeeds or retrieve all the Real Time events within a datafeed through Datafeed API.
+The Datafeed Service is a Core service built on top of the Datafeed API and provide a dedicated contract to bot developers to work with datafeed. 
+
+## Datafeed Configuration
+
+Datafeed Service can be configured by the datafeed field in the configuration file:
+
+```yaml
+datafeed:
+  version: 'v1' # or 'v2'
+  retry:
+    maxAttempts: 6
+    initialIntervalMillis: 2000
+    multiplier: 1.5
+    maxIntervalMillis: 10000
+```
+
+The minimal configuration for the datafeed service is the version of the datafeed which will be chosen to be use in the bdk.
+If the bot developers want to use the datafeed version 2 in their bot, the version configuration have to be specified as `v2`.
+Otherwise, the datafeed version 1 will be used by default.
+
+Bot developers can also configure a dedicated retry mechanism which will be used only by the datafeed service.
+Basically, the datafeed service retry configuration has the field same as the global retry configuration with the fields for implementing 
+the exponential backoff mechanism.
+
+## How to use
+The central component for the contract between bot developers and  the Datafeed API is the `DatafeedService`.
+This service is accessible from the `SymphonyBdk` object by calling the `datafeed()` method.
+For instance:
+
+```java
+import com.symphony.bdk.core.SymphonyBdk;
+public class Example {
+
+  public static void main(String[] args) {
+
+    final SymphonyBdk bdk = new SymphonyBdk(loadFromClasspath("config.yaml"));
+
+    final DatafeedService datafeedService = bdk.datafeed();
```

Comment:
I'd prefer to see a more concrete example, not just how to retrieve the `DatafeedService` that looks pretty obvious. 
Maybe just : 
```java
public class Example {

  public static void main(String[] args) throws Exception {

    final SymphonyBdk bdk = new SymphonyBdk(loadFromSymphonyDir("config.yaml"));    // (1)

    bdk.datafeed().subscribe(new RealTimeEventListener() {                                 // (2)

      @Override
      public void onMessageSent(V4Initiator initiator, V4MessageSent event) {
        bdk.messages().send(event.getMessage().getStream(), "<messageML>Hello, World!</messageML>");
      }
    });

    bdk.datafeed().start();                                                                // (3)
  }
}
``` 
is enough as a first example